mouse-click-effects@anaximeno/files/mouse-click-effects@anaximeno/5.4/settings-schema.json

```diff
@@ -49,7 +49,7 @@
 		},
 		"style-section": {
 			"type": "section",
-			"title": "Colors",
+			"title": "Style",
```

Comment:
Do you want to update the translation template with this PR or should it wait for another one?

Reply:
It's not completely necessary because 'Style' is also the title of that page so it's already set for translation. But it may impact the comments for the translation since it will be pointing to another line now.

Reply:
My bad for missing that.